stream/streams.go

```diff
@@ -165,17 +170,14 @@ type zipStream struct {
 }
 
 func (self zipStream) Next() (Event, error) {
```

Comment:
Put a comment here that this can cause resource leaks as we just drop all the streams except the first one, that's finished.
It's ok if all streams finish simultaneously, or if they don't hold a resource like goroutine or fd.